node/modules/services.go

```diff
@@ -58,7 +58,7 @@ func RunHello(mctx helpers.MetricsCtx, lc fx.Lifecycle, h host.Host, svc *hello.
 
 	sub, err := h.EventBus().Subscribe(new(event.EvtPeerIdentificationCompleted), eventbus.BufSize(1024))
 	if err != nil {
-		return xerrors.Errorf("failed to subscribe to event bus: %w", err)
+		return fmt.Errorf("failed to subscribe to event bus: %w", err)
```

Comment:
please dont replace this.  we use xerrors because it has stack traces.

Reply:
Done